src/components/EpisodesGrid/EpisodesGrid.js

```diff
@@ -29,9 +29,25 @@ class EpisodesGrid extends Component {
       });
   }
 
-  render() {
-    const { episodes, season } = this.state;
+  limititedSeason() {
+    let message = null;
+    const hasDefaultImages = this.state.episodes.every(({ image }) => image === null || image === sharkDefault);
+    if (hasDefaultImages) {
+      message = (
+        <div className="limitedDataWrapper">
+          <div className="limitedDataMessage">
+            <p>Limited data is available for Season {this.state.season}.</p>
+            <p>Check out Shark Week for more episodes!</p>
+            <a href="https://www.discovery.com/shark-week">Shark Week</a>
+          </div>
+        </div>
+      );
+    }
+    return message;
+  }
```

Comment:
You may be able to remove the message variable on line 33 by refactoring your code with the following. 

`limitedSeason() {
  const hasDefaultImages = episodes.every(({ image }) => image === null || image === sharkDefault);
  if (!hasDefaultImages) {
    return null;
  }
  return (
    <div className="limitedDataWrapper">
      <div className="limitedDataMessage">
        <p>Limited data is available for episodes in Season {season}.</p>
        <p>Check out <a className="shark-week-url" href="https://www.discovery.com/shark-week">Shark Week</a> for more details!</p>
      </div>
    </div>
  );
}`